library/datasort.h

```diff
@@ -81,8 +81,10 @@ struct datasort_cfg {
 	struct list_head		sorted_chunks;
 	/* Result of mergesort */
 	struct datasort_chunk		*result;
-	/* Datasort directory */
-	char				*dir;
+	/* Datasort directory - chunks are merged into single sorted blob here */
+	char				*datasort_dir;
+	/* Chunks directory - if not defined, then chunks are stored in datasort_dir */
+	char				*chunks_dir;
```

Comment:
The main purpose of #121 is to decrease load and space consumption from data disks.
If you put half of your load and consumption into separated disk and half will rest in datadir, this will not change situation alot. I'm not sure which part of sorting is more perf/space consuming, it was never tested.

I recommend to move everything related to sorting and defragmentation into separate dir which will be used just as a base dir for existing logic.